admin/tests/unit/controllers/edit-user.spec.js

```diff
@@ -159,6 +161,11 @@ describe('EditUserCtrl controller', () => {
   const mockContactGet = facilityId => {
     dbGet.resolves({ parent: { _id: facilityId } });
   };
+  const editUser = async () => {
+    await scope.editUser();
+    // Allow time for async validations to complete
+    await timeout.flush(2000);
```

Comment:
It seems like the additional Promise I added to the edit-user flow pushed the tests over the edge of some race-condition where previously the async validations were magically completing before the test tried to assert anything.

Now some assertions around the async validates were failing because the validates did not "have time" to execute.  With this flush, the async queue is cleared and everything can be properly asserted.